packages/common/types/command/PartialTargetDescriptor.types.ts

```diff
@@ -266,11 +266,11 @@ export interface InferPreviousMarkModifier {
   type: "inferPreviousMark";
 }
 
-export type Position = "before" | "after" | "start" | "end";
+export type TargetPosition = "before" | "after" | "start" | "end";
```

Comment:
Renamed because otherwise it conflicts with `Position` when we export it from `common/index.ts`